packages/web-components/src/components/va-accordion/va-accordion-item.scss

```diff
@@ -4,6 +4,37 @@
 
 @import '../../mixins/focusable.css';
 
+:host {
+  display: block;
+}
+
+:host(:not(:last-child)) {
+  margin-bottom: 0.5rem;
+}
+
+:host(:last-child) {
+  margin-bottom: 0.3125rem;
+}
+
+button {
+  font: inherit;
+  line-height: 1.15;
```

Comment:
This button `line-height` is applying from normalize.css in the USWDS component but we don't have that loading directly in the component-library (it loads in vets-website globally) so keeping that style here for now.